app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java

```diff
@@ -1092,4 +1104,14 @@ public void setWebAppsData(String aWebAppsData) {
     public String getWebAppsData() {
         return mPrefs.getString(mContext.getString(R.string.settings_key_web_apps_data), "");
     }
+
+    public void setEyeTrackingSupported(boolean isSupported) {
```

Comment:
So there are two different things here. First the runtime needs to support eye tracking, that's indeed checked by the DeviceDelegate. But then there is the option for the user to use it or not. We need to store that preference which is selected in the UI.

Reply:
But we are storing both, aren't we ? Otherwise, I think the names of the methods and status attributes are really confusing.

Reply:
Hmm no, we're only storing the user preference. The hardware support is always checked on initialization. We do store that value during runtime in an attribute of the DeviceDelegate but that's just to avoid checking it over and over again.

Reply:
Yes, you're right we store it to be able to show/hide elements in the UI. We don't really need to store that in the settings as you mention and perhaps it's indeed a bad choice. I'll come up with an alternative.

Reply:
Changed in the last version to a boolean in VRBrowserActivity